creating_images/sti.adoc

```diff
@@ -213,19 +213,21 @@ example:
 See the https://docs.docker.com/reference/builder/#onbuild[Docker documentation]
 for more information on `ONBUILD`.
 
-S2I has a different strategy when a Docker image with `ONBUILD` instructions is
-used as a builder image for the application source code. During the S2I build,
+Upon start S2I detects whether the builder image uses `ONBUILD` instructions.
+If there are none, the regular S2I link:#build-process[build] is performed,
+otherwise a different strategy is chosen. During such S2I build,
```

Comment:
Not sure about it, I'm talking here about one particular build process, that's why I used singular form.